src/main/java/org/fundacionjala/coding/digitalroot/DigitalRoot.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.coding.digitalroot;
```

Comment:
You should have your own package 
in this case william
Apply to all your pull requests